asistente_ladm_col/gui/supplies/dlg_missing_cobol_supplies.py

```diff
@@ -74,10 +75,11 @@ def accepted(self):
         self.save_settings(self.data_system)
 
         self.folder_path = self.txt_file_path_folder_supplies.text()
+        self.files_name = self.txt_files_name_supplies.text().strip()
         self.gpkg_path = os.path.join(self.folder_path, QCoreApplication.translate(
-                'MissingCobolSuppliesDialog', 'missing_supplies_cobol.gpkg'))
+                'MissingCobolSuppliesDialog', '{}.gpkg'.format(self.files_name)))
```

Comment:
If the file name is given by the user, there is no need to use `QCoreApplication.translate()`.

Reply:
Yes, I removed ```QCoreApplication.translate().``` from where it was not needed.